static/app/components/events/interfaces/crashContent/exception/lineCoverageLegend.tsx

```diff
@@ -1,33 +1,37 @@
 import styled from '@emotion/styled';
 
 import {Flex} from 'sentry/components/core/layout/flex';
+import {Text} from 'sentry/components/core/text/text';
 import {coverageText as COVERAGE_TEXT} from 'sentry/components/events/interfaces/frame/contextLineNumber';
 import {Coverage} from 'sentry/types/integrations';
 
 export function LineCoverageLegend() {
   return (
-    <Flex gap="2xl" align="center" direction="row" paddingBottom="md">
+    <Flex gap="2xl" align="center" direction="row" wrap="wrap">
       <CoveredLine>{COVERAGE_TEXT[Coverage.COVERED]}</CoveredLine>
       <UncoveredLine>{COVERAGE_TEXT[Coverage.NOT_COVERED]}</UncoveredLine>
       <PartiallyCoveredLine>{COVERAGE_TEXT[Coverage.PARTIAL]}</PartiallyCoveredLine>
     </Flex>
   );
 }
 
-const CoveredLine = styled('div')`
+const CoveredLine = styled(Text)`
```

Comment:
@calvin-codecov you should not be doing this, as it will at some point be removed. Font size and nowrap in this example should be set as Text props with the rest being set on the container element